crates/cheatcodes/src/inspector.rs

```diff
@@ -1419,15 +1421,59 @@ impl Inspector<&mut dyn DatabaseExt> for Cheatcodes {
         let should_check_emits = self
             .expected_emits
             .iter()
-            .any(|expected| expected.depth == ecx.journaled_state.depth()) &&
+            .any(|(expected, _count_map)| expected.depth == ecx.journaled_state.depth()) &&
```

Comment:
Style nit, prefer `(expected, _)`, not entirely clear if `_count_map` is used and prefixed because of name clashing with `count_map`